src/Confluent.Kafka/project.json

```diff
@@ -14,8 +14,7 @@
     },
 
     "frameworks": {
-        "net451": { },
```

Comment:
Yeah, that's messy, couldn't get it to work on Linux wiht that line still there.
It might be possible with build --framework 

Will revert that change.